From a0ad0bb5158e491a577c3fa0f0522fefc14e9f80 Mon Sep 17 00:00:00 2001
From: XenServer <xenserver@cloud.com>
Date: Fri, 20 Sep 2024 16:41:00 +0000
Subject: [PATCH 1/2] Sync vm.slice with xenserver-release-v8.4.0-12.tar.gz

  * Fri Sep 20 2024 Mark Syms <mark.syms@cloud.com> - 8.4.0-12
  - CA-399511: Change systemd dependencies for vm.slice

Signed-off-by: Yann Dirson <yann.dirson@vates.tech>
---
 src/common/etc/systemd/system/vm.slice | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/common/etc/systemd/system/vm.slice b/src/common/etc/systemd/system/vm.slice
index a5792eb..edf4cc2 100644
--- a/src/common/etc/systemd/system/vm.slice
+++ b/src/common/etc/systemd/system/vm.slice
@@ -1,8 +1,9 @@
 [Unit]
 Description=VM blkio slice
+Before=xapi-domains.service
 
 [Slice]
 BlockIOWeight=100
 
 [Install]
-WantedBy=multi-user.target
+RequiredBy=multi-user.target
-- 
2.39.5

